Repositories.Tests/HighlightRepositoryTest.cs

```diff
@@ -45,11 +43,7 @@ public class HighlightRepositoryTest : RepositoryTest<Highlight, HighlightReposi
         [Test]
```

Comment:
Empty returns tag (while this method does return something).

Reply:
It does not return it asserts.